src/coreclr/vm/siginfo.hpp

```diff
@@ -944,6 +944,32 @@ class MetaSig
         //------------------------------------------------------------------
         CorElementType GetByRefType(TypeHandle* pTy) const;
 
+        // Struct used to capture in/out state during the comparison
+        // of element types.
+        struct CompareState
+        {
+            // List of tokens that are currently being compared.
+            // See TokenPairList for more use details.
+            TokenPairList*  Visited;
+
+            // Custom modifiers found during the comparison.
+            DWORD CustomModifierCount1;
+            DWORD CustomModifierCount2;
```

Comment:
Delete - not used anymore?

Reply:
Eh. I was hoping to keep them to ensure a pattern is followed. Agreed they aren't used though.